manifests/zookeeper/README.md

```diff
@@ -0,0 +1,9 @@
+# How to install: 
+# Just execute by this way
+
+kubectl create -f namespace.yaml
+kubectl create -f zk.headless.svc.yaml
+kubectl create -f zk.svc.yaml
```

Comment:
Why is there a README in zookeeper dir, but not in kafka.

good practice to be consistent